pkg/cli/admin/release/bug.go

```diff
@@ -255,18 +255,10 @@ func GetBugList(b map[string]Bug) []Bug {
 	return bs
 }
 
-func generateBugKey(bt BugType, id int) string {
```

Comment:
not needed anymore because the jira ref IDs include the jira prefix now, so there's no risk of collision between a bugzilla bug "1234" and a jira bug "ocpbugs-1234"